src/cc_toolchain/wrapper/k8_gcc-cov

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash --norc
+
+exec /external/llvm_clang/bin/llvm-cov "$@"
```

Comment:
Probably shouldn't be calling `llvm-cov` here. We might need to also resolve #989 in this PR to get coverage working again? (@garethellis0 )

Reply:
can just be removed since gcc has coverage built in iinm